ts/DataGrid/Row.ts

```diff
@@ -172,6 +181,11 @@ abstract class Row {
             this.cells.splice(index, 1);
         }
     }
+
+    public setRowAttributes(): void {
+        const el = this.htmlElement;
+        el.setAttribute('aria-rowindex', this.index);
+    }
```

Comment:
Wouldn't it be better to move it directly to `HeaderRow`? In `TableRow` we have completely overridden this method and it is not used there.